src/main/java/com/exasol/containers/ExasolContainer.java

```diff
@@ -268,28 +270,54 @@ public T withClusterLogsPath(final Path clusterLogsHostPath) {
         return self();
     }
 
+    @Override
+    protected void containerIsStarting(final InspectContainerResponse containerInfo, final boolean reused) {
+        super.containerIsStarting(containerInfo, reused);
+        this.reused = reused;
+    }
```

Comment:
So there is option to reuse from testcontainers? That is great!
I was expecting we might have to do from scratch.

Reply:
Yes, I came across that, when I was trying to implement it myself ;-)